lib/ui/views/transactions/transaction_recent_list.dart

```diff
@@ -20,65 +20,42 @@ class TxList extends ConsumerWidget {
       ),
     );
 
-    final theme = ref.watch(ThemeProviders.selectedTheme);
-
     return Column(
       key: const Key('recentTransactions'),
       children: [
-        if (recentTransactions != null)
+        if (recentTransactions == null)
```

Comment:
Sorry, my request was not clear. The loading concerns the loading of the detail transaction sheet, not the list of transactions. You can remove this part because the loading of recent tx is managed by ArchethicRefreshIndicator in AccountTab widget